doc/publishing-finders.md

```diff
@@ -6,15 +6,9 @@ not fit the standard specialist document finder pattern. These are:
 - [Advanced search](advanced-search)
```

Comment:
The link to the advanced search docs doesn't work. I think it's missing a `.md`. Can you fix it as well?

Reply:
Fixed